### ISSUE TITLE
archive idea instead of deleting it - the commit is not packaged - possible to recover idea ?

### ISSUE THREAD

**Issue by memento** [simonv3/IdeaOtter#10]
Hi everyone,

Thanks for your work.
I use this application quite a lot and I wrote one of my business ideas, very detailed and refined. Once it was there and the next day it was gone. As I compared the code in my sandstorm app to the code on github, I discovered that the archiving process ([this commit](https://github.com/simonv3/IdeaOtter/commit/c17a6aa3fda8257ac64e100b9ede2994b02cc276)) wasn't yet in the package, which is too bad for me. 

My questions are: 
1) Do you think it's possible to recover that idea from the database (I think you use MongoDB which I've never used), or does no mean "NO" just like with the ladies ?
2) When do you think you could package the last version of your source code and make it available to upgrade in the sandstorm interface ?

Best regards,

**Comment by memento** [comment]
Anyone ?

**Comment by simonv3** [comment]
Hey @memento, I thought I had responded to this. Sorry! Glad to hear you use this app extensively :). 

I don't have a lot of time to fix bugs in idea otter at the moment (unfortunately :(), but I'm more than happy to accept pull requests.

Having said that here's the answers to your questions:

1) No. That item is most likely completely deleted unless there's a thing that Meteor does that I'm not aware of. It means "no" like with anyone ;). 
2) I thought there was already a packaged upload! I suspect it's already packaged and just needs to be sent to Sandstorm.

**Comment by ocdtrekkie** [comment]
Current Idea Otter on the app market is 1.4.0, current code version is 1.5.0.

**Comment by simonv3** [comment]
Yeah, I have been using 1.5.0 personally for a while, so I think I just forgot to publish it. Just did so and hopefully it'll get approved soon and you'll be able to use it!

**Comment by simonv3** [comment]
And I think you should now be able to update to the most recent version of ideaotter. 

However, looking at your initial description - it sounds like you had an idea just disappear? Did you accidentally delete it, or did it just _poof_ vanish? Poof vanish is worrying. 

This most recent commit archives ideas if you click on the "Delete" button, but if it vanishes then it's probably lost either way and we should figure out what went wrong.
